src/ui/component/channelEdit/view.jsx

```diff
@@ -5,6 +5,7 @@ import { Form, FormField } from 'component/common/form';
 import Button from 'component/button';
 
 import SelectAsset from '../selectAsset/view';
+import TagSelect from '../tagsSelect/view';
```

Comment:
These should be `component/selectAsset`/`component/tagsSelect`

@jessopb I think this might be why you were running into those flow issues, because you weren't importing the `index.js` file so no props will be injected.